### ISSUE TITLE
Serialization changes the original object

### ISSUE THREAD

**Issue by fcool** [neos/flow-development-collection#2011]
### Description

If you serialize an object containing related entities in a multivalued property (array or similar), bringing this object to serialization it will remove those entities from the object during serialization, which then will stop you from using the properties value after the serialization.

### Steps to Reproduce

See testcase in #2012 

### Affected Versions

Flow: 4.3 - 6.2

**Comment by fcool** [comment]
Let me know, if I should retarget for version 5.3. As the change is mergeable in all majors I targeted 4.3 as problems in serialization often leads to quite unexpected attac vectors.

**Comment by albe** [comment]
Yeah, I guess this also should be retargeted to 5.3 unless someone can find a security issue in overwriting the property?

**Comment by fcool** [comment]
I guess retargeting would be the easiest - if someone has concerns - he has still 6 months to find a reason for a backport.

I will come with a more detailed answer to your concern in the new pull request

**Comment by fcool** [comment]
It's this time again... as the fix has been reverted, it was a matter of time, when I will again run into this.
This is obviously today. :(
